.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+node_modules/
+forecastToken.js
```

Comment:
General note for your Procfile: It should have something in it, even though your server works right now without that content.  

Typically you would see: `web: node server.js` which tells the Heroku instance what process to run when it starts/restarts.